stun/stun.go

```diff
@@ -185,5 +190,9 @@ func doStunTransaction(lAddr, rAddr *net.UDPAddr) (publicIP net.IP, publicPort i
 	if err != nil {
 		logger.Log(1, "2:stun error: ", err.Error())
 	}
+	if err1 != nil {
+		logger.Log(3, "2:stun error: ", err1.Error())
```

Comment:
can we change the numbering on this log
logger.Log(3, "3:stun error: ", err1.Error())

Reply:
Yes, done in a  new commit